src/app-layout/classic.tsx

```diff
@@ -592,7 +592,7 @@ const ClassicAppLayout = React.forwardRef(
               bottomOffset={placement.insetBlockEnd}
               topOffset={placement.insetBlockStart}
               isMobile={isMobile}
-              drawers={drawers}
+              drawers={drawers.filter(item => !!item.trigger)}
```

Comment:
There is still one here: https://github.com/cloudscape-design/components/pull/2596#discussion_r1775056489